packages/builder/src/components/design/NavigationPanel/NewScreenModal.svelte

```diff
@@ -1,117 +1,117 @@
 <script>
-  import { store, allScreens, selectedAccessRole } from "builderStore"
+  import { store } from "builderStore"
   import { tables } from "stores/backend"
-  import { roles } from "stores/backend"
-  import { Input, Select, ModalContent, Toggle } from "@budibase/bbui"
+  import { ModalContent, Body, Detail, Layout, Icon } from "@budibase/bbui"
   import getTemplates from "builderStore/store/screenTemplates"
-  import analytics, { Events } from "analytics"
-  import sanitizeUrl from "builderStore/store/screenTemplates/utils/sanitizeUrl"
 
-  const CONTAINER = "@budibase/standard-components/container"
+  export let screenNameModal
+  export let navigationSelectionModal
+  export let selectedScreens = []
 
-  let name = ""
-  let routeError
-  let baseComponent = CONTAINER
-  let templateIndex
-  let draftScreen
-  let createLink = true
-  let roleId = $selectedAccessRole || "BASIC"
+  const blankScreen = "createFromScratch"
+  $: blankSelected = selectedScreens.find(x => x.id === blankScreen)
```

Comment:
Can this logic just be simplified to "is there one screen or not"? If you choose blank, then there's only one screen - if you chose one or more tables then there's always at least 3 screens right? I just saw this in a few places and thought we could simplify it.